docs/user-guide/measure.md

```diff
@@ -56,4 +57,16 @@ In the quadrant bearing system, the bearing of a line is measured as an angle fr
 
 The bearing measure will be reported as a tool tip and on the measure page.
 
-<p align = "center" ><img src="../img/measure-bear-1.jpg" style="max-width:600px;" /></p>
+<p align = "center" ><img src="../img/measure-bearing-1.png" style="max-width:600px;" /></p>
+
+Annotation
+------------------
+It is possible to define <a href='/user-guide/annotations'>annotation</a> from the measurements i.e length, area or bearing. 
```

Comment:
The link is not working, please use internal markdown references: something like 
`[annotation](annotations.md)`

Reply:
The link is not working, did you try it?